src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultDocumentContextFactory.cs

```diff
@@ -31,14 +31,25 @@ internal class DefaultDocumentContextFactory : DocumentContextFactory
         _logger = loggerFactory.CreateLogger<DefaultDocumentContextFactory>();
     }
 
-    public override async Task<DocumentContext?> TryCreateAsync(Uri documentUri, CancellationToken cancellationToken)
+    public override Task<DocumentContext?> TryCreateAsync(Uri documentUri, CancellationToken cancellationToken)
+     => TryCreateCoreAsync(documentUri, versioned: false, cancellationToken);
```

Comment:
You `.ConfigureAwait(false)` below, do you need to do it here too?

Reply:
You can only configure an await, if there is in fact an await :)

Reply:
lol yep that'll do it.